src/consumer/config.rs

```diff
@@ -22,6 +22,8 @@ pub struct ConsumerConfig {
     pub(crate) consumer_name: Option<String>,
     /// numerical id of the consumer
     pub(crate) consumer_id: Option<u64>,
+    /// size of the receiver queue
+    pub(crate) receiver_queue_size: Option<u32>,
```

Comment:
This field should be added to `ConsumerOptions`, then we don't need to add a new `receiver_queue_size` method

Reply:
Done